modules/custom/openy_daxko2/src/Plugin/migrate/process/OpenYDaxkoSchedule.php

```diff
@@ -20,19 +20,54 @@ class OpenYDaxkoSchedule extends ProcessPluginBase {
    * {@inheritdoc}
    */
   public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    if (empty(json_decode($value))) {
+      return [];
+    }
     $value = json_decode($value);
 //    {"times":[{"start":"18:00","end":"20:00"}],"days":[{"id":"5","name":"Friday"}],"start_date":"2018-02-16T00:00:00.0000000","end_date":"2018-02-16T00:00:00.0000000"}
 
-    $startDate = substr($value->start_date, 0, 11) . $value->times[0]->start . ':00';
-    $endDate = substr($value->end_date, 0, 11) . $value->times[0]->end . ':00';
+    // Check for max timestamp date
+    $timezone = new \DateTimeZone(drupal_get_user_timezone());
+    $maxDt = new \DateTime();
+    $maxDt->setTimestamp(2147483647);
```

Comment:
Let's make this as a constant, it is better to avoid hardcoded variables in the code